src/tests/system/tests/test_ldap.py

```diff
@@ -56,7 +57,8 @@ def test_ldap__change_password(client: Client, ldap: LDAP, modify_mode: str):
 
 
 @pytest.mark.ticket(bz=[795044, 1695574])
-@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify"])
+@pytest.mark.importance("critical")
```

Comment:
These are those unwarranted changes I was talking about in [MR ldap_login_grace](https://github.com/SSSD/sssd/pull/7722#issuecomment-2505647103). sssd-2-9 and master have 'critical' as importance markers. Whereas sssd-2-9-4 (rhel-8-10) has 'authentication' marker. 
These kind of changes are not even related to the main MR.

Reply:
These are those unwarranted changes I was talking about in [MR ldap_login_grace](https://github.com/SSSD/sssd/pull/7722#issuecomment-2505647103). sssd-2-9 and master have 'critical' as importance markers. Whereas sssd-2-9-4 (rhel-8-10) has 'authentication' marker. 
These kind of changes are not even related to the main MR.

Reply:
That' why instead of cherry picking the MR, I chose to backport manually in [https://github.com/SSSD/sssd/pull/7722](https://github.com/SSSD/sssd/pull/7722)